### ISSUE TITLE
Small amount of bleed

### ISSUE THREAD

**Issue by pmjm** [ZFTurbo/MVSEP-MDX23-music-separation-model#8]
Truly an excellent implementation of stem separation! 

But I believe there is a problem in the mixing. Seems there is a small amount of bleed in some stems from the others. One good way to hear this is trying separation on the album version of Bon Jovi's "You Give Love A Bad Name". You can hear the opening acapella in all the stems, and the result is amplified significantly in the instrumental mix when all those are summed together.

Likewise, the vocal stem has a low level of the sum of the other stems mixed in it as well. 

I don't think this is an issue of identifying frequencies for a stem, rather a mixing issue perhaps when using complementary phase cancellation.

Cheers!

**Comment by Dyslexicon** [comment]
The example youve cited here with bleed on a vocal acapella is likely another symptom of the vocal stem issues I submitted in Issue#2. This vocal stem model, while better at distinguishing guitars from vocals than htdemucs_ft, it is also prone to more bleed in areas without vocals, resulting in non-vocal frequencies being assigned to the vocal stem.

To be fair this model is astonishingly clean of bleed (as well as spectral holes), in comparison with any model I've heard or analyzed the output of (except for the vocal model, as I have mentioned in Issue#2).
The more examples I render the more it becomes apparent how much an improvement this model is over htdemucs_ft.
I am stunned by how much better and fuller Drums, Bass, and "Other" sound than htdemucs_ft.

If the lossy-trained vocal model is simply replaced with a full-spectrum clean vocal model, minor issues in the other stems would likewise be mitigated accordingly. The best of both worlds would be if the cleanliness and full-spectrum response of htdemucs_ft vocal model could be combined with the better identification distinction between vocals and guitar found on this model.

**Comment by whamer100** [comment]
Yeah there's definitely something odd going on with bleeding, it feels like its just playing the original song at 1% volume behind the split stems.

**Comment by kubinka0505** [comment]
> Yeah there's definitely something odd going on with bleeding, it feels like its just playing the original song at 1% volume behind the split stems.

resep with htdemucs then 🙂
